### ISSUE TITLE
until operator

### ISSUE THREAD

**Issue by safazi** [pigpigyyy/Yuescript#52]
`until` is the `unless` equivalent for `while`.

Example:
```coffee
a = 3
until a == 0
    a -= 1
```

Could generate
```lua
a = 3
while not (a == 0) do
    a = a - 1
end
```

**Comment by nikneym** [comment]
I think it's not possible this way since `until` is reserved for `repeat` - `until` loop.

**Comment by pigpigyyy** [comment]
This syntax is OK with repeat-until loop and I've got it implemented with 4d61a00ebc5b956da72525de0e180de28a1e8ac6

**Comment by safazi** [comment]
well done!
